nistats/contrasts.py

```diff
@@ -0,0 +1,241 @@
+import numpy as np
```

Comment:
Please remove all the corresponding code from glm.py. I am quite anxious about code duplication.